clusters/app.ci/openshift-image-registry/domain.md

```diff
@@ -0,0 +1,85 @@
+# Expose registry.ci.openshift.org as registry's domain name
+
+After the registry's `defaultRoute` is enabled, its domain name is
+
+```
+oc --context app.ci get route -n openshift-image-registry
```

Comment:
Use `console` highlight for some bells an whistles :)

````
 ```console
 $ do stuff
 stuff
 ```
````
->

```console
$ do
stuff
```

Reply:
Updated